packages/xo-server/src/xo-mixins/backups-ng-logs.js

```diff
@@ -49,10 +49,15 @@ const taskTimeComparator = ({ start: s1, end: e1 }, { start: s2, end: e2 }) => {
 
 export default {
   async getBackupNgLogs (runId?: string) {
-    const { runningJobs } = this
+    const { runningJobs, runningRestores } = this
     const consolidated = {}
     const started = {}
-    forEach(await this.getLogs('jobs'), ({ data, time, message }, id) => {
+    const [jobLogs, restoreLogs] = await Promise.all([
+      this.getLogs('jobs'),
+      this.getLogs('restore'),
+    ])
+
+    forEach({ ...jobLogs, ...restoreLogs }, ({ data, time, message }, id) => {
```

Comment:
Concatenating the log is a bad idea performance-wise, simply run the `forEach` twice.